File: game-utils/default-bots/java/src/core/api/InitialData.java
package core.api;

import core.MatchDetails;

import java.util.HashMap;

public class InitialData extends HashMap<String, Object> {
    // TODO remove extends HashMap and add fields specific to your game

    // TODO If you provided your own implementation for the game, uncomment this:
    // public MatchDetails[] __matchDetails;

    // TODO If you provided your own implementation for the game then create
    //      normal __uid field and return it here
    public int getUid() {
        return (int) (double) get("__uid");
    }
}

File: game-utils/game-example/bots/kotlin/src/core/api/InitialData.kt
package core.api

import java.util.HashMap

class InitialData(
        // TODO If you provided your own implementation for the game, uncomment this:
        // var __matchDetails: Array<MatchDetails>
) : HashMap<String, Any>() {
    // TODO remove extends HashMap and add fields specific to your game

    // TODO If you provided your own implementation for the game then create
    //      normal __uid field and return it here
    fun getUid(): Int {
        return (get("__uid") as Double).toInt()
    }
}
